# Goal definition
geometry_msgs/PoseStamped goal_pose
string target_map
---
# Result definition
bool success
---
# Feedback definition
string status
